### ISSUE TITLE
Terms from NANoREG harmonised terminology 

### ISSUE THREAD

**Issue by vedina** [enanomapper/ontologies#77]
There are 43 terms and corresponding definitions in Table 3.1 of 2016 document
`NANoREG harmonised terminology for environmental health and safety assessment of nanomaterials` 
https://ec.europa.eu/jrc/en/publication/nanoreg-harmonised-terminology-environmental-health-and-safety-assessment-nanomaterials

Would it be possible to add the terms with exact definitions as per Table 3.1 ? Linking to existing terms in the ontology would be good as well.

**Comment by JKChang2015** [comment]
Yes, those terms in T3.1 could be defined using RDF schema _rdfs:isDefinedBy_ . Otherwise, I can create a new annotation property _NANoREG DEFINITION_ for it.
